auacm/app/modules/blog_manager/blog_test.py

```diff
@@ -35,59 +33,57 @@ def testCreate(self):
                 self.assertEqual(post[key], post_response[key])
 
         # Delete the post from the test database
-        session.flush()
-        post = session.query(BlogPost).filter(
-            BlogPost.id == post_response['id']).first()
-        session.delete(post)
-        session.commit()
+        DATABASE_SESSION.flush()
+        post = DATABASE_SESSION.query(BlogPost).filter(
+            BlogPost.post_id == post_response['id']).first()
+        DATABASE_SESSION.delete(post)
+        DATABASE_SESSION.commit()
 
-    def testGetAll(self):
```

Comment:
Correct me if this has changed, but when this code was written the canonical convention was to use camelCase for unit test names